app/models/filterspam.rb

```diff
@@ -0,0 +1,21 @@
+class Filterspam < ActiveRecord::Base
+  has_many :comments
+  has_many :likes
+  belongs_to :user
+
+  has_attached_file :image, :styles => { :medium => "300x300" }, :default_url => "/images/:style/missing.png"
```

Comment:
Metrics/LineLength: Line is too long. [109/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)